src/Uno.UWP/ApplicationModel/Package.Other.cs

```diff
@@ -21,9 +21,9 @@ public partial class Package
 		private string _logo = "ms-appx://logo";
 		private bool _manifestParsed;
 
-		private bool GetInnerIsDevelopmentMode() => false;
+		private bool GetIsDevelopmentMode() => false;
 
-		private DateTimeOffset GetInstallDate() => DateTimeOffset.Now;
+		private DateTimeOffset GetInstallDate() => new DateTimeOffset(2000, 1, 1, 0, 0, 0, TimeSpan.Zero);
```

Comment:
I wonder if we could create a file (or read some other file) on first launch then read the date so we can provide that date more reliably?

Reply:
That is a good idea! It would be the first launch date, but that is still more accurate than nothing 😀 . However, first we need to resolve https://github.com/unoplatform/uno/issues/8910 so that it is really app specific.

Reply:
Why use such placeholders? It is simply not supported atrribute.
App programmer should be warned on compile time that he/she should check app logic that uses GetInstalledDate.
E.g.: tryware, and logic as 
` if(date.Now - GetInstalledDate > 14 days) showerror("free use period ends, please buy this app")`